engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetViewportToSizeOf.java

```diff
@@ -95,7 +81,6 @@ public String toString() { // TODO: used for logging purposes at the moment, inv
     }
 
     private FBO getFbo() {
-        return defaultDynamicFBO != null ? defaultDynamicFBO.getFbo() : frameBuffersManager.get(fboName);
-
+        return frameBuffersManager.get(fboName);
```

Comment:
They are hidden in the PR because I did not edit them.
Four of them are in equals() for accessing the current and the other fbo, two of them in update() that you see.